docker/1.6.0/py2/Dockerfile.gpu

```diff
@@ -129,6 +129,9 @@ RUN pip install --no-cache-dir --upgrade \
     python-dateutil==2.8.0 \
     mpi4py==3.0.2 \
     ${MX_URL} \
+    # inotify-simple updated to 1.3.0 and has an issue that prevents the installation
```

Comment:
Nit: You can simply write inotify-simple >=1.3.0 has an issue.

Reply:
Wanted to provide details so that this constraint can either be fixed better or removed in the next iteration.